ktxspec.adoc

```diff
@@ -871,35 +871,43 @@ A _dfDescriptor_ is useful in the following cases:
 The following restrictions must be obeyed when setting the fields of a
 _dfDescriptorBlock_.
 
-* The value of `<<_vkformat,vkFormat>>` and the image data must
-  never contradict the DFD and, vice-versa, the DFD must not
-  contradict `<<_vkformat,vkFormat>>`. Thus the value of `vkFormat`
-  can be used as a shortcut. Only when it is `VK_FORMAT_UNDEFINED`,
-  does an application need to utilitize sample information from the
-  DFD.  (In this case contradiction is impossible.)
-* `colorModel` must be `KHR_DF_MODEL_RGBSDA`
-  or one of the block compressed color models listed in <<KDF13>>
-  Section 5.6 or its successors, currently `KHR_DF_MODEL_BC1A` to
-  `KHR_DF_MODEL_UASTC`.
-* `colorPrimaries` may be any of the available values since
-  conversion of the selected primaries and white point to a display's
-  can be done simply with a 3x3 matrix multiply.
+* If `<<_vkformat,vkFormat>>` is not `VK_FORMAT_UNDEFINED`, the DFD's
+  `texelBlockDimension*`, `bytesPlane*`, and sample information fields
```

Comment:
s/, and/and/. Sorry just noticed this. I haven't used oxford commas anywhere else in the spec., at least not intentionally, so please delete this one.

Reply:
Looks like both styles are used in the text. FWIW, Vulkan style guide requires it.

Reply:
Okay. Leave it. I was unaware of the Vulkan style guide and I am surprised to see how many oxford commas there are in the spec. as it is not my writing style. The link from the Vulkan style guide to the Oxford Dictionaries' blog of the video discussion of said commas doesn't work any more.